pkgs/test/pubspec.yaml

```diff
@@ -8,34 +8,41 @@ environment:
 
 dependencies:
   analyzer: '>=0.36.0 <0.42.0'
```

Comment:
Can this get widened at the same time?

https://github.com/dart-lang/sdk/blob/master/pkg/analyzer/pubspec.yaml